app/src/main/java/com/sun/chat_04/data/repositories/UserDataSource.kt

```diff
@@ -17,5 +17,9 @@ interface UserDataSource {
         fun loginByEmailAndPassword(email: String, password: String, callback: RemoteCallback<Boolean>)
 
         fun upgradeLocationUser(location: Location, callback: RemoteCallback<Boolean>)
+
+        fun getUsers(callback: RemoteCallback<List<User>>)
+
+        fun getInfoUser(callback: RemoteCallback<User>)
```

Comment:
```suggestion
        fun getUserInfo(callback: RemoteCallback<User>)
```
getInfo mà k biết get của user nào sao em?